com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.cs

```diff
@@ -852,66 +852,135 @@ static void OnLightingDataCleared()
             Clear();
         }
 
-        static void WriteToShaderCoeffsL0L1(ref SphericalHarmonicsL2 sh, NativeArray<float> shaderCoeffsL0L1, int offset)
+        static ushort SHFloatToHalf(float value)
+        {
+            return Mathf.FloatToHalf(value);
```

Comment:
does it clamp if value is too big for 16 bit, or does it overflow ?

Reply:
Unity Doc seems to imply it's clamped.